common/visualization.html

```diff
@@ -31,51 +31,21 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <link rel="shortcut icon" type="image/png" href="logo16.png"/>
+        <link rel="shortcut icon" type="image/png" href="logo16.png">
+        <link rel="stylesheet" type="text/css" href="style.css">
         <title>OpenSeadragon</title>
-        <style type="text/css">
-
-            body {
-                margin: 0;
-                background-color: black;
-                color: white;
-                text-align: center;
-                font-size: x-large;
-            }
-
-            #loader {
-                position: absolute;
-                top: 0;
-                left: 0;
-                bottom: 0;
-                right: 0;
-            }
-
-            #loader img {
-                vertical-align: middle;
-                max-width: 100%;
-                max-height: 100%;
-            }
-
-            .img-vertical-align-helper {
-                display: inline-block;
-                height: 100%;
-                vertical-align: middle;
-            }
-
-            #openseadragon {
-                position: absolute;
-                top: 0;
-                left: 0;
-                bottom: 0;
-                right: 0;
-            }
-
-        </style>
     </head>
     <body>
         <div id="openseadragon"></div>
         <div id="loader"><span class="img-vertical-align-helper"></span></div>
+        <div id="popup">
+            <div id="error"></div>
+            <label for="url">Enter the URL of an image to visualize:</label><br>
+            <input type="url" id="url">
+            <button type="button" onclick="OpenSeadragonizer.open();">
+                Show!
+            </button>
+        </div>
```

Comment:
Maybe a small "powered by OpenSeadragon" link back to the main site should go in here?